VARIABLES.md

```diff
@@ -1720,7 +1768,7 @@ Below are the supported variables for the role confluent.kafka_broker
 
 ### kafka_broker_custom_log4j
 
-Boolean to enable cp-ansible's Custom Log4j Configuration
+Boolean to reconfigure Kafka's logging with RollingFileAppender and log cleanup
```

Comment:
Boolean to reconfigure the Kafka Broker's logging with the RollingFileAppender and log cleanup functionality.